src/net.cpp

```diff
@@ -3552,6 +3552,13 @@ std::vector<CAddress> CConnman::GetAddresses(CNode& requestor, size_t max_addres
         // nodes to be "terrible" (see IsTerrible()) if the timestamps are older than 30 days,
         // max. 24 hours of "penalty" due to cache shouldn't make any meaningful difference
         // in terms of the freshness of the response.
+
```

Comment:
I think we should put here a long comment explaining why we're setting all the timestamps to the same fixed value, similar to the comment above explaining why it's useful to have a cache